src/lib/components/organisms/ticker/style.module.scss

```diff
@@ -81,6 +81,7 @@
 .button {
   min-height: 40px;
   background-color: var(--tertiary-bg-color);
+  padding-bottom: 20px;
```

Comment:
added this to give the button more space from the bottom of the screen at mobile